plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TestTransactionLogTail.java

```diff
@@ -66,9 +63,8 @@ private List<DeltaLakeTransactionLogEntry> readJsonTransactionLogTails(String ta
             throws Exception
     {
         TrinoFileSystem fileSystem = new HdfsFileSystemFactory(HDFS_ENVIRONMENT).create(SESSION);
-        URI resource = getClass().getClassLoader().getResource(tableLocation).toURI();
-        Path tablePath = new Path(resource);
-        TransactionLogTail transactionLogTail = TransactionLogTail.loadNewTail(fileSystem, tablePath, Optional.of(10L));
+        String tableLocationFromResources = getClass().getClassLoader().getResource(tableLocation).toURI().toString();
+        TransactionLogTail transactionLogTail = TransactionLogTail.loadNewTail(fileSystem, tableLocationFromResources, Optional.of(10L));
```

Comment:
nit: We can build the resource's location within `testTail` and then simplify `updateJsonTransactionLogTails` & `readJsonTransactionLogTails` methods.
```java
String tableLocation = getClass().getClassLoader().getResource(format("%s/person", dataSource)).toURI().toString();
```